the-collab-lab/tcl-68-smart-shopping-list
# Install Tailwind

## Description

- Installs Tailwind

## Related Issue

Closes #42 

## Acceptance Criteria

- [X] Tailwind is present in package.json as a dependency

## Type of Changes

`enhancement`

## Updates

### Before

![Screenshot 2024-03-28 at 10 56 57 AM](https://github.com/the-collab-lab/tcl-68-smart-shopping-list/assets/7997357/2a45a5f6-21e7-4d04-9f7c-73c20ad0e252)

### After

![Screenshot 2024-03-28 at 10 56 15 AM](https://github.com/the-collab-lab/tcl-68-smart-shopping-list/assets/7997357/c3890b9d-26d7-47de-b75b-158a25a2c90e)

## Testing Steps / QA Criteria

- Checkout branch ```et-install-tailwind``` (I think it might be necessary or at least helpful to run locally for this particular PR)
- Run ```npm install``` in the project directory
- Run ```npm start```
- Confirm that the styling looks like the "After" screenshot
- Add a Tailwind className to an element and confirm that the styling changes appropriately
